src/vcpkg/dependencies.cpp

```diff
@@ -1177,51 +1181,47 @@ namespace vcpkg
 
         if (!excluded.empty())
         {
-            print2("The following packages are excluded:\n", actions_to_output_string(excluded), '\n');
+            msg::println(msg::format(msgExcludedPackages).append_raw("\n" + actions_to_output_string(excluded)));
         }
 
         if (!already_installed_plans.empty())
         {
-            print2("The following packages are already installed:\n",
-                   actions_to_output_string(already_installed_plans),
-                   '\n');
+            msg::println(
+                msg::format(msgInstalledPackages).append_raw("\n" + actions_to_output_string(already_installed_plans)));
         }
 
         if (!remove_specs.empty())
         {
-            std::string msg = "The following packages will be removed:\n";
+            auto message = msg::format(msgPackagesToRemove).append_raw("\n");
```

Comment:
```suggestion
            auto message = msg::format(msgPackagesToRemove);
```

This `\n` is already added by either `append_raw` or `println`. Please check that the output here makes sense by building vcpkg and running with various install and remove commands to trigger the conditions, and compare with the 'before' state. Example:

![image](https://user-images.githubusercontent.com/1544943/180331540-f1b453dc-b469-413c-aec6-828b1a80f000.png)

(I think this change will insert an extra newline before the actual list...)